Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docker.io/splunk/splunk@sha256:0bcd9c9f836cfb72b80b5ff8deec692b561473e68b9e41ea5c1f7d92abc35c61
+
+ENV SPLUNK_START_ARGS=--accept-license
+
+EXPOSE 8089 8000
+
+USER root
```

Comment:
We should switch back to a non-root user later on. Looking at the Splunk example in the segment-bridge repo, the user is most likely `ansible`.